locustfile.py

```diff
@@ -0,0 +1,26 @@
+from locust import HttpUser, task, between
+
+
+class ChatUser(HttpUser):
+    wait_time = between(60,61)
+
+    def on_start(self):
+        self.client.get("/")
+
+    @task
+    def ask_question(self):
+        self.client.post("/chat", json={
+            "history": [
+                {
+                "user": "What is included in my Northwind Health Plus plan that is not in standard?"
```

Comment:
I'd suggest having a list of questions and using random.choice instead of the same question each time.